.golangci.yml

```diff
@@ -3,6 +3,7 @@
 #   timeout: 5m
 
 linters:
+  disable-all: true
```

Comment:
how come?

Reply:
Goland was complaining about `--disable-errcheck` and `--disable-all` being passed together. I think that it's due to some internal config of the plugin.

However, having both linters white and black lists doesn't make much sense. Considering how golangci-lint seems to be running in our environments (I believe it does run with `--disable-all` on), it makes sense to just make the operational mode explicit. In other words:

1. disable all
2. enable only specific linters

I welcome suggestions though!

Reply:
>Goland was complaining about --disable-errcheck and --disable-all being passed together. I think that it's due to some internal config of the plugin.

it sounds like you have some configuration wrong in your env, in vs code you can do this: 
```
  "go.lintTool": "golangci-lint",
  "go.lintFlags": [
    "--fast",
  ],
```
can you not do this in goland? 

`disable-all:true` should only be disabling errcheck, a linter that should be enabled. Just want to make sure we aren't disabling something that will cause something to slip past us

Reply:
This is the error message I receive:

```shell
$ golangci-lint run --disable-all --disable errcheck
ERRO Running error: can't combine options --disable-all and --disable errcheck
```

The error makes sense cause `--disable-all` (as well as respective `disable-all: true` config value) disable all linters, not just `errcheck`.

So we should either set `disable-all:true` then populate the `enable` linterns list, or `disable-all:false` and then populate the `disable` linters list.

Reply:
I'm fine either way. I was just interested to see how this was happening. we can merge as-is. 😄